handlers/messages.go

```diff
@@ -40,6 +40,15 @@ func MessageHandler(session *discordgo.Session, msg *discordgo.MessageCreate) {
 		case "echo": // echo the message that was sent
 			log.Println("msghandler: caught echo command")
 			echo(session, msg, args)
+		
+		case "glizzy": // glizzy
```

Comment:
I would consider refactoring messages architecture. One of the signs of good architecture is that adding new functionality means adding new code, rather than editing existing code.

Rather than a switch block you could have a way to register Keywords with their implementation. For example having a RemindMeKeyword.go which implements a handle func and a match func. MessageHandler could then loop each registered Keyword and if it matches run it's handle and return. That breaks up responsibilities better, rather than one func doing many things. The parsing of commands is a separate concern from the implementation of each keyword.

Also reviewers should approve before code is merged :/